Svc/SeqDispatcher/test/ut/SeqDispatcherTesterHelpers.cpp

```diff
@@ -0,0 +1,58 @@
+// ======================================================================
```

Comment:
If you did not make changes to this file, I'd recommend adding:

`set(UT_AUTO_HELPERS ON)` to your CMakeList.txt and deleting this file.

Reply:
Ah, I actually never knew what that UT_AUTO_HELPERS was doing. Now I understand!! So many questions about where the init and connectPorts methods were coming from are answered :)